index.html

```diff
@@ -3,47 +3,279 @@
+    {"State":"North Dakota","Region":"Midwest","RepVotes":188163.00,"TotalVotes":322645.00,"perc":0.583188954,"Percentage":"58.31%"},
+    {"State":"Ohio","Region":"Midwest","RepVotes":2661407.00,"TotalVotes":5580822.00,"perc":0.476884409,"Percentage":"47.68%"},
+    {"State":"Oklahoma","Region":"South","RepVotes":891325.00,"TotalVotes":1334872.00,"perc":0.667723197,"Percentage":"66.77%"},
+    {"State":"Oregon","Region":"West","RepVotes":754175.00,"TotalVotes":1789270.00,"perc":0.421498712,"Percentage":"42.14%"},
+    {"State":"Pennsylvania","Region":"Northeast","RepVotes":2680434.00,"TotalVotes":5753670.00,"perc":0.465865091,"Percentage":"46.58%"},
+    {"State":"Rhode Island","Region":"Northeast","RepVotes":157204.00,"TotalVotes":446049.00,"perc":0.352436616,"Percentage":"35.24%"},
+    {"State":"South Carolina","Region":"South","RepVotes":1071645.00,"TotalVotes":1964118.00,"perc":0.545611313,"Percentage":"54.56%"},
+    {"State":"South Dakota","Region":"Midwest","RepVotes":210610.00,"TotalVotes":363815.00,"perc":0.578893119,"Percentage":"57.88%"},
+    {"State":"Tennessee","Region":"South","RepVotes":1462330.00,"TotalVotes":2458577.00,"perc":0.594787147,"Percentage":"59.47%"},
+    {"State":"Texas","Region":"South","RepVotes":4569843.00,"TotalVotes":7993851.00,"perc":0.571669775,"Percentage":"57.16%"},
+    {"State":"Utah","Region":"West","RepVotes":740600.00,"TotalVotes":1017440.00,"perc":0.727905331,"Percentage":"72.79%"},
+    {"State":"Vermont","Region":"Northeast","RepVotes":92698.00,"TotalVotes":299290.00,"perc":0.309726352,"Percentage":"30.97%"},
+    {"State":"Virginia","Region":"South","RepVotes":1822522.00,"TotalVotes":3854489.00,"perc":0.472831029,"Percentage":"47.28%"},
+    {"State":"Washington","Region":"West","RepVotes":1290670.00,"TotalVotes":3125516.00,"perc":0.412946214,"Percentage":"41.29%"},
+    {"State":"West Virginia","Region":"South","RepVotes":417655.00,"TotalVotes":670438.00,"perc":0.622958424,"Percentage":"62.29%"},
+    {"State":"Wisconsin","Region":"Midwest","RepVotes":1407966.00,"TotalVotes":3068488.00,"perc":0.458846833,"Percentage":"45.88%"},
+    {"State":"Wyoming","Region":"West","RepVotes":170962.00,"TotalVotes":249061.00,"perc":0.686426217,"Percentage":"68.64%"}
+  ];
+
+  var MAX_PERCENT = 0.728; //Set the maximum value for bars
+
+  var options = {
+    sorted: false,
+    filtered: false
+  };
+
+  function updateTable() {
+    var data = voting.slice();
+    if (options.sorted) {
+      if (options.sorted === 'asc') {
+        data.sort(function (a, b) { return a.perc - b.perc; });
+      } else if (option.sorted === 'desc') {
+        data.sort(function (a, b) { return b.perc - a.perc; });
+      }
+    }
+
+  // Filter by region of the country (west)
+      if(options.filtered) {    
+        if (options.filtered === 'region-w') {
+           data = data.filter(function (c) {
+              return c.Region === "West"; });
+        } 
+    }
```

Comment:
So these could all be nested inside a single `if (options.filtered) {}`.

For next week, wonder if you could rewrite this using the `object[key]` syntax we showed in class, as opposed to the dot syntax (`object.key`) used here. The end result could be simpler, particularly if the capitalization of options.filtered matches the data.

I also want to note on the column names that are used in your data (like "Region"), I generally advocate for using Javascript naming conventions (camelCase), just so you don't have to switch back and forth. That being said, snake_case is also acceptable and commonly used; Javascript coding styles do, however, avoid the use of variables that start with capital letters unless they are all-caps CONSTANTS, unchanging values like `TOTAL_DELEGATES`, or a [constructor function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new).